cmd/server/assets/users/index.html

```diff
@@ -19,69 +19,74 @@
     {{template "flash" .}}
 
     <h1>Users</h1>
-    <p>
-      These are the users of this realm. You can also <a href="/users/new">create a new user</a>.
-    </p>
+    <p>Manage users of <strong>{{.currentRealm.Name}}</strong>.</p>
 
-    {{if $users}}
-    <div class="table-responsive">
-      <table class="table table-bordered table-striped bg-white">
-        <thead>
-          <tr>
-            <th scope="col">Name</th>
-            <th scope="col" width="300">Email</th>
-            <th scope="col" width="75">Options</th>
-            <th scope="col" width="40"></th>
-          </tr>
-        </thead>
-        <tbody>
-          {{range $users}}
-          <tr>
-            <td>
-              <a href="/users/{{.ID}}">{{.Name}}</a>
-            </td>
-            <td>
-              {{.Email}}
-            </td>
-            <td class="text-center">
-              {{if .CanAdminRealm $currentRealm.ID}}
-              <span class="badge badge-pill badge-primary">Admin</span>
+    <div class="card mb-3 shadow-sm" style="min-width:550px;">
+      <div class="card-header">Users</div>
+      <div class="card-body">
+        <a class=" btn btn-primary" href="/users/new">Create a new user</a>
+        <a class="btn btn-primary disabled" href="/users/import">Import users</a>
+        {{if $users}}
+        <div class="table-responsive mt-3">
```

Comment:
We generally want all margins to be on the _bottom_ for consistency. Can you move this mt to be a mb on the preceeding element?